actionpack/lib/action_dispatch/middleware/show_exceptions.rb

```diff
@@ -80,6 +81,14 @@ def fallback_to_html_format_if_invalid_mime_type(request)
         end
       end
 
+      def fallback_to_html_when_invalid_multipart(request)
+        rack_error = request.get_header(Rack::RACK_REQUEST_FORM_ERROR)
+        return unless rack_error.is_a?(::EOFError)
+
+        request.delete_header(Rack::RACK_REQUEST_FORM_ERROR)
+        request.set_header "CONTENT_TYPE", "text/html"
```

Comment:
`text/html` doesn't make much sense as a request `Content-Type`. 

If we need some fallback value, I think `application/octet-stream` would make more sense, no?

Reply:
Thanks, yeah good point.